frontend/quasar.conf.js

```diff
@@ -75,6 +75,13 @@ module.exports = configure(function (ctx) {
           });
         }
       },
+      extendWebpack(cfg) {
+        cfg.module.rules.push({
+          resourceQuery: /blockType=i18n/,
+          type: 'javascript/auto',
+          use: [{ loader: '@kazupon/vue-i18n-loader' }, { loader: 'yaml-loader' }],
```

Comment:
This is where the yaml-loader is used following the guide here https://v1.quasar.dev/options/app-internationalization#setting-up-translation-blocks-in-your-sfcs